### PR TITLE
Fix log depth z fighting

### LINKED ISSUE #6508: Log depth z fighting
Steps to reproduce:

* Zoom in close to the polygon
* Zoom out to the initial starting view
* Start rotating the camera

[localhost:8080 Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=ZVNtT9swEP4rVr80SMHNq5NAQRsUaUhsQ1BNmpZ9MM4lsebYle2UdRP/fU7S8jKsKPKd77nn7vF5SzXacngEjc6QhEd0CYb3Hf42+rw5G81LJS3lEvT86LSUpdw6lIbqVoldo6RDTikwSMstB4NpVXl/S4nckrQDdILmd1ChzR7gPtPrmjKY+1PU4eQE7WHDajloqlm7c+59XZdUW7ejMsa1Vt0KGg1gPmpNd96P4zBMceCjOBv+z1Y0WcHojeO9NXrj8I3l4n8e+S8FdNSC5lS84ldCaXx3tZqCnkr5tJdkUsAwkICZ61lTbMAOOh6UqMBYLqnlSp681vqlJ+84yYskymIcpyQroiIiQ3VRnJOYFDhOsighhEQ+ygOSRHmB0yxNQ0KSIj3UrTR31/Ce5RPQisvmllvW3ikhvADnQRIkWRAneZoGjmPgwmk4kEVhGhTE7WIfERxFSZCFJEjDKE9jcqACWa01laZWunuR6DO1mv9O8PXq6sv6ev39WaJ7KitGjRUwzMdaNY2Ai95aJb35jWqcPhvbzn1kdQ8+qnvJhiY81gL7BdXRYTLeKC1UQzW3bcfZaoBf9HU9zvIedTqyz/zZ0tidgPMpxQfebZS2qNfCw3hhodsId9Vm8dA7kMXMmKHk5eIAWlZ8i3h1Vs7+exDlDDFBjXEndS/EPf8D5ex8uXDxb2BCjep/3YIWdDeEtOH5zeTEGC8XznyPskqJB6pfZfwH)

flicker-log-depth

### PR DESCRIPTION
**Pull request by bagnell**
Fixes #6508.

This was caused by #6465. This enables writing log depth to gl_Position when the camera is far enough away.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@bagnell, thanks for the pull request! Maintainers, we have a signed CLA from @bagnell, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by mramato** [comment]
@lilleyse can you please review this ASAP Monday morning, I don't want to merge any last minute PRs on Tuesday.

File: Source/Shaders/Builtin/Functions/vertexLogDepth.glsl [hunk 1]
```diff
@@ -7,6 +7,13 @@ void czm_updatePositionDepth() {
 #if defined(LOG_DEPTH) && !defined(DISABLE_GL_POSITION_LOG_DEPTH)
     v_logPositionEC = (czm_inverseProjection * gl_Position).xyz;
 
+#ifdef ENABLE_GL_POSITION_LOG_DEPTH_AT_HEIGHT
+    if (length(v_logPositionEC) < 2.0e6)
```
Comment (lilleyse) [inline Source/Shaders/Builtin/Functions/vertexLogDepth.glsl]: The fix works... I'm just wondering if there is anything special about `2.0e6` or if it just happens to be a good value.

Comment (bagnell) [reply Source/Shaders/Builtin/Functions/vertexLogDepth.glsl]: I zoomed in to the edge of the polygon and slowly zoomed out until I saw the z fighting. Tested with larger and smaller polygons than what was in the bug example.

Comment (lilleyse) [reply Source/Shaders/Builtin/Functions/vertexLogDepth.glsl]: Alright, thanks!

**Pull request merged**